security/restricting-fiddler-features.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Restricting Fiddler
+description: "Using the Enterprise policy feature to disable specific Fiddler Everywhere features like remote connections, changing ports, cloud services, and others."
+slug: fe-restrict-policies
+publish: true
+position: 40
+---
+
+# Restricting Fiddler
+
+Users of Fiddler Everywhere often work in a secure corporate environment where specific policies are imposed. That means that administrators need to have options to limit or hardcode particular features of the Fiddler Everywhere application. With the Fiddler Everywhere Enterprise tier, administrators can control a set of functionalities through policy management systems on macOS and Windows.
+
+## Restricting Features on macOS
+
+
+Users on macOS with administrative privileges set policies through the [**defaults**](https://macos-defaults.com/) command and by using the `com.progress-telerik.fiddler` domain alongside the following policies:
+
+| Policy          | Description | Value Type |  Value to apply the policy
+|:-----------------|:----------------|:----------------|:-----------------
+| **DisableCloud**   | Disables sharing options  | integer  | 1
```

Comment:
Disables all interactions with the Fiddler Everywhere's cloud - this includes sharing, storing data in the cloud and receiving data which was shared with the current user through our cloud.
Important note here is that users will still need access to Fiddler Everywhere's cloud endpoints for login and license validation purpose.